_posts/2023-04-04-the-rails-foundation-welcomes-cedarcode-as-a-contributing-member.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The Rails Foundation welcomes Cedarcode as a Contributing member"
+categories: news
+author: The Rails Foundation
+published: true
+date: 2023-04-03
+---
+
+Beginning this week, we are excited to announce that the Rails Foundation will be regularly updating this blog with information on the initiatives we are actively working on. Stay tuned for the latest updates on our progress and how we are working to improve the Ruby on Rails community.
+
+First up: The Rails Foundation recently welcomed its first Contributing member: <a href="https://www.cedarcode.com/">Cedarcode</a>.
+
+<img src="/assets/images/logo-cedarcode.svg">
+
+Cedarcode integrates smoothly with companies to collaborate full-time on their development projects and help them boost their teams. With many members of the team bringing more than 10 years of Ruby on Rails experience to their work, Cedarcode is well equipped to offer high-quality code, attention to detail, and a passion for performance and security to a wide range of businesses and industries.
+
+In addition to their expertise in Ruby on Rails development, Cedarcode is also committed to giving back to the community. They have organized and participated in numerous community events, have contributed to open-source projects, including the Rails framework itself, as well as created and maintain <a href="https://www.cedarcode.com/open-source_">their own open-source projects</a>.
```

Comment:
There is an `_` at the end of the URL which makes it lead to a 404 page.

Reply:
Thanks @gregmolnar - fixed!